opensrp-app/src/main/java/org/smartregister/repository/PlanDefinitionSearchRepository.java

```diff
@@ -86,6 +94,33 @@ public void addOrUpdate(PlanDefinition planDefinition, String jurisdiction) {
         return getPlanDefinitionRepository().findPlanDefinitionByIds(planIds);
     }
 
+    public List<PlanDefinitionSearch> findPlanDefinitionSearchByPlanId(@NonNull String planId) {
+        List<PlanDefinitionSearch> planDefinitionSearchSet = new ArrayList<>();
+        String query = String.format("SELECT " + StringUtils.join(COLUMNS, ",") + " FROM %s " +
+                "WHERE %s=? ", PLAN_DEFINITION_SEARCH_TABLE, PLAN_ID);
```

Comment:
Since you are using format, can you remove the string concatenation

Reply:
updated